packages/nemo-evaluator-launcher/examples/slurm_llama_3_1_8b_instruct.yaml

```diff
@@ -35,11 +35,12 @@ defaults:
 
 # override default execution arguments
 execution:
-  account: ???
-  output_dir: /some/filesystem/path/available/to/slurm/node
+  hostname: ???          # SLURM headnode hostname (required)
+  username: ???          # Optional: defaults to ${oc.env:USER} from slurm/default
```

Comment:
Let's not override the default to ??? here, it works for most users. Maybe we could add the same value as default here just for the sake of showing this param to the users?